File: CH1-input-process-output/EX3-land-calculation/source-code/landCalculation.cpp
#include <iostream>
using namespace std;

int main()
{
    int squareFoot;
    int acre;

    // Input
    cout << "Enter a Square Foot :";
    cin >> squareFoot;

    // Process
    acre = squareFoot / 43560;

    // Output
    cout << "acre :" << acre << endl;

    return 0;
}

File: CH1-input-process-output/EX3-land-calculation/source-code/landCalculation.java
import java.util.Scanner;

public class landCalculation {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        float squareFeet;
        int acre;

        System.out.print("Enter a Square feet : ");
        squareFeet = input.nextFloat();

        double temp = squareFeet / 43.560;
        acre = (int) temp;

        System.out.print("Acres : " + acre);

        input.close();
    }

}